geomesa-tools/conf/application.conf

```diff
@@ -1,7 +1,6 @@
-geomesa = {
-  sfts = [
-    {
-      type-name = "example-csv"
```

Comment:
interestingly enough i guess the config name could be techincally different from the SFT type name...i think we want to get away from it all together...but you do have the weird case where you provide it as an arg by itself...we could force people to nest it properly in ```geomesa.sfts.<name>``` in that case which might be best.